src/main/java/com/marklogic/kafka/connect/source/AbstractRowBatchBuilder.java

```diff
@@ -37,7 +40,7 @@ void logBatchError(Exception ex) {
      * @param parsedConfig - The complete configuration object including any transform parameters.
      * @param rowBatcher - The RowBatcher object to be configured.
      */
-    protected void configureRowBatcher(Map<String, Object> parsedConfig, RowBatcher<?> rowBatcher) {
+    protected void configureRowBatcher(Map<String, Object> parsedConfig, RowBatcher<?> rowBatcher, String previousBatchMaxValue) {
```

Comment:
I know this is verbose, but I think it's important that this param be clearly associated with the constraining column, so I'm thinking `previousMaxConstraintColumnValue`. Again, that's wordy, but I'm not sure how to shorten it (it's a simpler name if we assume like Kafka JDBC does that it's always a timestamp, but we know that won't necessarily be the case).